src/test/unit/motor_output_unittest.cc

```diff
@@ -28,6 +28,41 @@ extern "C" {
 #include "unittest_macros.h"
 #include "gtest/gtest.h"
 
+extern "C" {
+	uint8_t atomic_BASEPRI;
+
+	bool featureIsEnabled(uint8_t f);
+	float getDigitalIdleOffset(void *cfg);
+	float scaleRangef(float a, float b, float c, float d, float e);
+
+}
+
+// Mocking functions
+
+bool featureIsEnabled(uint8_t f)
+{
+	(void)f;
+	return true;
+}
+
+float getDigitalIdleOffset(void *cfg)
+{
+	(void)cfg;
+	return 0;
+}
+
+float scaleRangef(float a, float b, float c, float d, float e)
+{
+	(void)a;
+	(void)b;
+	(void)c;
+	(void)d;
+	(void)e;
+	return 0;
+}
+
```

Comment:
```suggestion

}
```

This wraps the whole segment (including the stubs) into the "extern" macro.

Reply:
fixed